File: relationship-manager/auth-service/src/main/java/com/bank/rm/auth/AuthServiceApplication.java
package com.bank.rm.auth;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.bank.rm.auth", "com.bank.rm.common"})
public class AuthServiceApplication {
    public static void main(String[] args) {
        SpringApplication.run(AuthServiceApplication.class, args);
    }
}

File: relationship-manager/auth-service/src/main/kotlin/com/bank/rm/auth/AuthServiceApplication.kt
package com.bank.rm.auth

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.bank.rm.auth", "com.bank.rm.common"])
class AuthServiceApplication

fun main(args: Array<String>) {
    runApplication<AuthServiceApplication>(*args)
}
